plugins/plugin-testing-java/plugin-testing-base/src/main/java/org/eclipse/che/plugin/java/testing/JavaTestAnnotations.java

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
This class should be an enum.

Reply:
This is redundant class, i've just removed it